web/src/js/components/register/index.js

```diff
@@ -41,7 +41,7 @@ export default class Register extends React.Component {
         return <div className="form-group">
             <div className="input-field col-1-1 no-height">
                 <label className="input-header" htmlFor="food_concerns">{this.context.translate('Er det noe mat du/dere ikke spiser')}?</label><br />
-                <input type="text" placeholder="Fyll inn" id="food_concerns" ref={(c) => this.form.food_concerns = c} />
+                <input type="text" placeholder="" id="food_concerns" ref={(c) => this.form.food_concerns = c} />
```

Comment:
Does the localisation not support placeholders?

Reply:
@jimbofreedman it does, however didn't make much sense to me. It simply says - enter.
Let me know if you think otherwise

Reply:
No you're right that doesn't really add much